plugins/python-build/bin/python-build

```diff
@@ -397,7 +397,7 @@ http_get_aria2c() {
   fi
 
   if aria2c --allow-overwrite=true --no-conf=true -d "${dir_out}" -o "${out}" ${ARIA2_OPTS} "$1" >&4; then
-    [ -n "$2" ] || cat "${out}"
+    [ -n "$2" ] || cat "${dir_out}/${out}"
```

Comment:
This would never `cat` the right file. I suppose no one else is using `http get` on systems with aria2c and without an output filename 😉